src/Microsoft.Management.Configuration.Processor/DscModules/DscModuleV2.cs

```diff
@@ -216,6 +216,16 @@ public IReadOnlyList<DscResourceInfoInternal> GetAllDscResources(PowerShell pwsh
             return setResult?.RebootRequired;
         }
 
+        /// <inheritdoc/>
+        public IList<ValueSet> InvokeExportResource(

```

Comment:
Again RE: interface.  Almost all (or all) of the changes to the processor changes could be removed with the interface semantic change.

I think that the DSC v3 integration does not benefit from being in .Net, so I don't know that any of this needs to be here when that comes either.